lib/checks/aria/aria-allowed-attr.json

```
@@ -1,6 +1,9 @@
 {
   "id": "aria-allowed-attr",
   "evaluate": "aria-allowed-attr-evaluate",
+  "options": {
+    "ariaAttr": ["aria-posinset", "aria-setsize", "aria-expanded", "aria-level"]
```

This needs a better name. How about:

```suggestion
    "invalidRowAttrs": ["aria-posinset", "aria-setsize", "aria-expanded", "aria-level"]
```

---

I agree